etc/config/fortran.amazon.properties

```diff
@@ -729,6 +729,18 @@ compiler.flangtrunk-fc.supportsBinaryObject=false
 compiler.flangtrunk-fc.supportsBinaryExecute=false
 compiler.flangtrunk-fc.alias=flangtrunknew-fc
 
+#################################
+# LFortran
+group.lfortran.compilers=lfortran
+group.lfortran.groupName=LFortran
+
+compiler.lfortran.name=lfortran (latest)
+compiler.lfortran.exe=/opt/compiler-explorer/lfortran/bin/lfortran
```

Comment:
In order to support more fortrans I'm going to merge this and then modify it (will CC you on the changes fyi).